cscore/src/main/java/edu/wpi/first/cscore/CvSink.java

```diff
@@ -41,8 +42,8 @@ public CvSink(String name) {
    * @param image Where to store the image.
    * @return Frame time, or 0 on error (call GetError() to obtain the error message)
    */
-  public long grabFrame(Mat image) {
-    return grabFrame(image, 0.225);
+  public long grabFrame(Mat image, PixelFormat pixelFormat) {
```

Comment:
API wise, I think we should make the pixelFormat an argument on the CvSink constructor instead of passing it to grabFrame.  The pixel format really shouldn't change on a frame-to-frame basis.  Also, we should retain the current constructor to provide a BGR default so this change is non-breaking.